client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -121,8 +135,27 @@ class DomainRegistrationSuggestion extends React.Component {
 					  } )
 					: translate( 'Select', { context: 'Domain mapping suggestion button' } );
 		}
+
+		let buttonStyles = { primary: true };
+		if ( abtest( 'domainSearchButtonStyles' ) === 'onePrimary' ) {
+			buttonStyles = ! isFeatured ? {} : this.props.buttonStyles;
+		}
+
+		if ( this.isUnavailableDomain( suggestion.domain_name ) ) {
+			buttonStyles = { ...buttonStyles, disabled: true };
+			buttonContent = translate( 'Unavailable', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Post unavailable' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/pre-check-domain-availability-again)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?